packages/common/src/major2-validation.ts

```diff
@@ -79,9 +80,43 @@ type AndError = {
     | {
         type: typeof MajorValidationErrorType.And.NoSolution;
         discoveredAtChild: number;
+      }
+    | {
+        type: typeof MajorValidationErrorType.And.UnsatChildAndNoSolution;
+        noSolution: {
+          type: typeof MajorValidationErrorType.And.NoSolution;
+          discoveredAtChild: number;
+        };
+        unsatChildErrors: {
+          type: typeof MajorValidationErrorType.And.UnsatChild;
+          childErrors: Array<ChildError>;
+        };
```

Comment:
This is kind of a gross type, but matches the other types and is something we can take another look at after alpha.